src/main/java/org/elasticsearch/cluster/block/ClusterBlockLevel.java

```diff
@@ -54,34 +45,14 @@ public int id() {
         return this.id;
     }
 
-    /**
-     * Returns the ClusterBlockLevel's id according to a given version, this to ensure backward compatibility.
-     *
-     * @param version the version
-     * @return the ClusterBlockLevel's id
-     */
-    public int toId(Version version) {
-        assert version != null : "Version shouldn't be null";
-        // Since 1.6.0, METADATA has been split into two distincts cluster block levels
-        if (version.before(Version.V_1_6_0)) {
-            if (this == ClusterBlockLevel.METADATA_READ || this == ClusterBlockLevel.METADATA_WRITE) {
-                return ClusterBlockLevel.METADATA.id();
-            }
-        }
-        return id();
-    }
-
     static EnumSet<ClusterBlockLevel> fromId(int id) {
```

Comment:
I think the `fromId` doesn't need to return an `EnumSet` anymore, we can go back to just `ClusterBlockLevel`. Thoughts?